types/angular-block-ui/index.d.ts

```diff
@@ -70,7 +70,7 @@ declare module 'angular' {
              * @param {angular.IRequestConfig} config - the Angular request config object.
```

Comment:
Some day I'd like to go through everything and remove redundant JSDoc type annotations.